### PR TITLE
chore(files_versions): Refactor function for lisibility

### PR DESCRIPTION
**Pull request by solracsf**
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by kesselb** [comment]
I don't consider that an improvement :see_no_evil: 

Fine by me, but please fix the commit message ;)

**Comment by solracsf** [comment]
What's wrong with the commit message? 🤔

**Comment by kesselb** [comment]
The commit type should be "refactoring" instead of "chore" https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type.

lisbility => legibility or readability

**Review by provokateurin — COMMENTED** [review]
I agree with @kesselb, it's not really better than before.
It also abuses the match functionality, so I don't really like that either.

File: apps/files_versions/lib/Listener/FileEventsListener.php [hunk 1]
```diff
@@ -331,7 +302,7 @@ public function pre_renameOrCopy_hook(Node $source, Node $target): void {
 		$manager = Filesystem::getMountManager();
 		$mount = $manager->find($absOldPath);
 		$internalPath = $mount->getInternalPath($absOldPath);
-		if ($internalPath === '' and $mount instanceof MoveableMount) {
+		if ($internalPath === '' && $mount instanceof MoveableMount) {
```
Comment (susnux) [inline apps/files_versions/lib/Listener/FileEventsListener.php:305]: We should have a cs fixer rule for this

Comment (provokateurin) [reply apps/files_versions/lib/Listener/FileEventsListener.php:305]: Rector does this as well, I've never seen anyone use this syntax in recent years so it is only in legacy code anyway.

**Comment by come-nc** [comment]
I agree with the others, I dislike using match like this. It’s harder to read as the calls are not aligned with each other anymore.
Closing. The and thing is not big enough to justify a PR.

**Pull request closed without merging**